test/float-ops.2asm.js

```diff
@@ -0,0 +1,294 @@
+function asmFunc(global, env, buffer) {
+ "use asm";
+ var HEAP8 = new global.Int8Array(buffer);
+ var HEAP16 = new global.Int16Array(buffer);
+ var HEAP32 = new global.Int32Array(buffer);
+ var HEAPU8 = new global.Uint8Array(buffer);
+ var HEAPU16 = new global.Uint16Array(buffer);
+ var HEAPU32 = new global.Uint32Array(buffer);
+ var HEAPF32 = new global.Float32Array(buffer);
+ var HEAPF64 = new global.Float64Array(buffer);
+ var Math_imul = global.Math.imul;
+ var Math_fround = global.Math.fround;
+ var Math_abs = global.Math.abs;
+ var Math_clz32 = global.Math.clz32;
+ function dummy() {
+  
+ }
+ 
+ function $$1($$0, $$1) {
```

Comment:
btw, it would be nice to have wasm2asm emit proper function names (same name as in the wasm)

Reply:
For ease of debugging?

Reply:
Yeah, I think it would help debugging and also reading the testcases.

Reply:
Maybe we should only do it with `-g` or something like that?